config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -284,7 +284,7 @@ presubmits:
     decorate: true
     branches:
     - master
-    - release-1.8
+    - release-1.9
```

Comment:
Suggestion: we don't support k8s 1.18 any more, we can remove this e2e

Reply:
Thanks, I have removed it in 5bd46dc.